debian_versions.bzl

```diff
@@ -259,14 +259,14 @@ DEBIAN_PACKAGE_VERSIONS = {
             "libffi6": "3.2.1-9",
             "libgcc1": "1:8.3.0-6",
             "libgomp1": "8.3.0-6",
-            "liblzma5": "5.2.4-1+deb10u1",
+            "liblzma5": "5.2.4-1",
             "libreadline7": "7.0-5",
             "libsqlite3-0": "3.27.2-3+deb10u1",
-            "libssl1.1": "1.1.1n-0+deb10u3",
+            "libssl1.1": "1.1.1n-0+deb10u1",
             "libstdcpp6": "8.3.0-6",
             "mime-support": "3.62",
             "netbase": "5.6",
-            "openssl": "1.1.1n-0+deb10u3",
+            "openssl": "1.1.1n-0+deb10u1",
```

Comment:
this is because of the lack of security indexes on s390x and ppc64le on debian10